Connect with Google tutorial (#190)
sora-xor/sora-docs: src/polkaswap-connect-with-google.md

![](.gitbook/assets/google-importing-4.png)

5. After you input your account password, it will be available to select within the “Connect With Google” screen.

![](.gitbook/assets/google-importing-5.png)

## Making Transactions with a Google Account

After your Google account has been selected as the active account, your balances will be available in the Wallet view.

![](.gitbook/assets/google-transaction-1.png)

1. When swapping with your Google account, the process remains the same. Select the assets to swap, confirm, and confirm the fees.

![](.gitbook/assets/google-transaction-2.png)

2. The Polkaswap interface will request your password to sign the transaction for your Google account, keeping it as safe as any dApp signer. You can also set Polkaswap to remember your password for fifteen minutes in _[Substrate Style](create-an-address.md#via-polkadot-js-browser-plugin)_.

![](.gitbook/assets/google-transaction-3.png)

The same will apply to liquidity provision transactions, both double (_pooling_) and single-sided (_staking_)—as well as Bridge transactions and claiming rewards.

> ### Read More: [Liquidity Operations on Polkaswap](/provide-liquidity-to-xyk-pools-polkaswap), [Swaps on Polkaswap](/swap-polkaswap)

Reviewer: ```suggestion
## Learn More
- [Liquidity Operations on Polkaswap](/provide-liquidity-to-xyk-pools-polkaswap)
- [Swaps on Polkaswap](/swap-polkaswap)
```

---
title: "Polkaswap Google Login Tutorial | Securely Connect Your Account"
head:
  - - meta
    - name: description

>       content: "Learn how to securely connect your Polkaswap account with Google in this step-by-step tutorial. Follow our guide to create a new account, import an existing one using a JSON file, and make secure transactions with your Google account. Ensure the safety of your assets on Polkaswap while enjoying style and freedom in your transactions."

Reviewer: ```suggestion
      content: "Learn how to securely connect your Polkaswap account with a Google account in this step-by-step tutorial. Follow our guide to create a new account, import an existing one using a JSON file, and make secure transactions with your Google account. Ensure the safety of your assets on Polkaswap while enjoying style and freedom in your transactions."
```

  - - meta
    - name: keywords
      content: "Polkaswap Google login, Polkaswap account, Google account, Polkaswap tutorial, connect account, Google login tutorial, SORA account, import account, JSON file, transaction security"
---

# Connect With Google

> Connecting to [Polkaswap.io ](https://polkaswap.io/#/wallet) has never been easier. Use a Google login to access your SORA account with style and freedom.

Reviewer: ```suggestion
Connecting to [Polkaswap.io](https://polkaswap.io/#/wallet) has never been easier. Use a Google login to access your SORA account with style and freedom.
```

> This feature is for people who are new at DeFi and blockchain accounts.

Reviewer: maybe something along those lines?

```suggestion
This feature would be of most interest to people who are new at DeFi and blockchain accounts.
```

> **Never share your personal account information with anyone**.

Reviewer: this should be a warning

```suggestion
::: warning
Never share your personal account information with anyone.
:::
```

![](.gitbook/assets/google-login-1.png)

> 1. Navigate to the Account section in [Polkaswap.io ](https://polkaswap.io/#/wallet) and click the “Switch Account” button.

Reviewer: but also you don't need to link to polkaswap everytime you mention it

```suggestion
1. Navigate to the Account section in [Polkaswap.io](https://polkaswap.io/#/wallet) and click the “Switch Account” button.
```

![](.gitbook/assets/google-login-2.png)

2. Click on Google and sign in with the existing account on your computer or create a new account.

![](.gitbook/assets/google-login-3.png)

> 3. Allow [Polkaswap.io ](http://Polkaswap.io) access to your Google account, after allowing this permission, Polkaswap will use the keys to your account, safely stored within your Google Drive.

Reviewer: again, you don't need  to link to the website every time you mention polkaswap

```suggestion
3. Allow [Polkaswap.io](http://Polkaswap.io) access to your Google account, after allowing this permission, Polkaswap will use the keys to your account, safely stored within your Google Drive.
```

![](.gitbook/assets/google-login-4.png)

4. Once you have successfully signed in, you can create a new account or import an account using a JSON file.

![](.gitbook/assets/google-login-5.png)

5. If you want to create a new account, click the button and store the 12-word phrase to secure your account, also known as seed or [mnemonic phrase](create-an-address.md#restoring-account-from-mnemonic-in-different-applications). Store it safely, and never share it with anyone!

![](.gitbook/assets/google-login-6.png)

> 6. Confirm your passphrase to secure your account, or skip the process. (You can return to confirm your passphrase later.

Reviewer: ```suggestion
6. Confirm your passphrase to secure your account, or skip the process. You can return to confirm your passphrase later.
```

![](.gitbook/assets/google-login-7.png)

7. Give your account a name and password, then confirm the password. You can also export a JSON file at this point. **If you toggle the JSON file for the account restoration option, Polkaswap will download your account’s JSON file to your computer or device.**

::: info
You can use the JSON file to import an account on Polkaswap or another wallet.
:::

![](.gitbook/assets/google-login-8.png)

Success! You have created a SORA account to use on Polkaswap using Google!

![](.gitbook/assets/google-importing-4.png)

To access Polkaswap using this account, click on the account’s name, then input the account password you set in the last step.

**_You are now connected through Google with style and freedom!_**

## Importing an Existing Google Account

> 1. With the JSON file of the account you want to import easy to reach, navigate to the Account section on Polkaswap and click the Switch account button.

Reviewer: `to import easy to reach` -- what do you mean? please rephrase

Author: ```suggestion
1. With the JSON file of the account you want to import at hand, navigate to the Account section on Polkaswap and click the Switch account button.
```

Author: @outoftardis Is this better phrasing? I am trying to convey that the reader should have the JSON file ready to drag and drop

Reviewer: yes, this is better

![](.gitbook/assets/google-importing-1.png)

2. Click on Google and sign in to the account you want to use. You will find a screen with options to create or import an account if you have already signed in. Select the **Import Account** option.

![](.gitbook/assets/google-importing-2.png)

3. Here you can drag and drop the JSON file for your account, and there are also tutorials to help you export your JSON file from [Fearless

>    Wallet](https://wiki.fearlesswallet.io/accounts/walkthrough/exporting-and-importing-a-wallet-using-a-json-file), [Polkadot JS](https://support.polkadot.network/support/solutions/articles/65000177677-how-to-export-your-json-backup-file), and [Subwallet.](https://docs.subwallet.app/extension-user-guide/export-and-backup-an-account)

Reviewer: ```suggestion
   Wallet](https://wiki.fearlesswallet.io/accounts/walkthrough/exporting-and-importing-a-wallet-using-a-json-file), [Polkadot JS](https://support.polkadot.network/support/solutions/articles/65000177677-how-to-export-your-json-backup-file), and [Subwallet](https://docs.subwallet.app/extension-user-guide/export-and-backup-an-account).
```

![](.gitbook/assets/google-importing-4.png)

5. After you input your account password, it will be available to select within the “Connect With Google” screen.

![](.gitbook/assets/google-importing-5.png)

## Making Transactions with a Google Account

After your Google account has been selected as the active account, your balances will be available in the Wallet view.

![](.gitbook/assets/google-transaction-1.png)

1. When swapping with your Google account, the process remains the same. Select the assets to swap, confirm, and confirm the fees.

![](.gitbook/assets/google-transaction-2.png)

2. The Polkaswap interface will request your password to sign the transaction for your Google account, keeping it as safe as any dApp signer. You can also set Polkaswap to remember your password for fifteen minutes in _[Substrate Style](create-an-address.md#via-polkadot-js-browser-plugin)_.

![](.gitbook/assets/google-transaction-3.png)

The same will apply to liquidity provision transactions, both double (_pooling_) and single-sided (_staking_)—as well as Bridge transactions and claiming rewards.

> ### Read More: [Liquidity Operations on Polkaswap](https://wiki.sora.org/provide-liquidity-to-xyk-pools-polkaswap.html), [Swaps on Polkaswap](https://wiki.sora.org/swap-polkaswap.html)

Reviewer 2: ```suggestion
### Read More: [Liquidity Operations on Polkaswap](/provide-liquidity-to-xyk-pools-polkaswap), [Swaps on Polkaswap](/swap-polkaswap)
```

> ### Read More: [Liquidity Operations on Polkaswap](https://wiki.sora.org/provide-liquidity-to-xyk-pools-polkaswap.html), [Swaps on Polkaswap](https://wiki.sora.org/swap-polkaswap.html)

Reviewer: ```suggestion
## Learn More
- [Liquidity Operations on Polkaswap](https://wiki.sora.org/provide-liquidity-to-xyk-pools-polkaswap.html)
- [Swaps on Polkaswap](https://wiki.sora.org/swap-polkaswap.html)
```

Author: @outoftardis it seems like this is outdated and I cannot commit

Author: Also, I will be using this format to address [189](https://github.com/sora-xor/sora-docs/issues/189) 
Thank you.

Author: Actually, does it make sense to link them as pages of the wiki internally, instead of external links? So use "swap-polkaswap.md" instead of the current link?
